_posts/2021-10-25-waku-v1-vs-waku-v2.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: post
+name:  "Waku v1 vs Waku v2: bandwidth comparison"
+title:  "Waku v1 vs Waku v2: bandwidth comparison"
+date:   2021-10-25 17:00:00 +0200
+author: hanno
+published: true
+permalink: /waku-v1-vs-waku-v2
+categories: research
+summary: A local comparison of the bandwidth profile between Waku v1 and Waku v2 for similar network scenarios.
```

Comment:
I'd include conclusion here to make it more "tweetable"

Reply:
Done in 21749a4. Feel free to check if it's punchy enough.